src/View/Helper/AbstractHelper.php

```diff
@@ -51,14 +51,15 @@ abstract class AbstractHelper extends BaseAbstractHelper
      * @var array
      */
     protected $booleanAttributes = [
-        'autofocus' => ['on' => 'autofocus', 'off' => ''], // https://html.spec.whatwg.org/#attr-fe-autofocus
-        'checked'   => ['on' => 'checked',   'off' => ''], // https://html.spec.whatwg.org/#attr-input-checked
-        'disabled'  => ['on' => 'disabled',  'off' => ''], // https://html.spec.whatwg.org/#attr-fe-disabled
-        'itemscope' => ['on' => 'itemscope', 'off' => ''], // https://html.spec.whatwg.org/#attr-itemscope
-        'multiple'  => ['on' => 'multiple',  'off' => ''], // https://html.spec.whatwg.org/#attr-input-multiple
-        'readonly'  => ['on' => 'readonly',  'off' => ''], // https://html.spec.whatwg.org/#attr-input-readonly
-        'required'  => ['on' => 'required',  'off' => ''], // https://html.spec.whatwg.org/#attr-input-required
-        'selected'  => ['on' => 'selected',  'off' => ''], // https://html.spec.whatwg.org/#attr-option-selected
+        'autofocus' =>  ['on' => 'autofocus',  'off' => ''], // https://html.spec.whatwg.org/#attr-fe-autofocus
+        'checked'   =>  ['on' => 'checked',    'off' => ''], // https://html.spec.whatwg.org/#attr-input-checked
+        'disabled'  =>  ['on' => 'disabled',   'off' => ''], // https://html.spec.whatwg.org/#attr-fe-disabled
+        'itemscope' =>  ['on' => 'itemscope',  'off' => ''], // https://html.spec.whatwg.org/#attr-itemscope
+        'multiple'  =>  ['on' => 'multiple',   'off' => ''], // https://html.spec.whatwg.org/#attr-input-multiple
+        'readonly'  =>  ['on' => 'readonly',   'off' => ''], // https://html.spec.whatwg.org/#attr-input-readonly
+        'required'  =>  ['on' => 'required',   'off' => ''], // https://html.spec.whatwg.org/#attr-input-required
+        'selected'  =>  ['on' => 'selected',   'off' => ''], // https://html.spec.whatwg.org/#attr-option-selected
+        'novalidate' => ['on' => 'novalidate', 'off' => ''], // https://html.spec.whatwg.org/#attr-option-selected
```

Comment:
Adding `novalidate` here is correct, but everything else must be reverted. See also the automated checks.

Reply:
@froschdesign  I have reverted all but my change but the automated checks are still failing

Reply:
@krytenuk - The array arrows have to line up as per the coding standard. You can run `vendor/bin/phpcbf` and it will fix it for you. In this case, the BC-Check failure will continue to fail and you can't change that.

Reply:
I tried to run `vendor/bin/phpcbf` locally but it was complaining about `LaminasCodingStandard`. Re-ran `composer install` and now working. Have pushed changes.